src/Streamly/Internal/Data/Tee.hs

```diff
@@ -0,0 +1,53 @@
+-- |
+-- Module      : Streamly.Internal.Data.Tee
```

Comment:
This should be `Streamly.Internal.Data.Fold.Tee`, the way we have `Streamly.Internal.Data.Stream.Zip`.